packages/typespec-azure-portal-core/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@azure-tools/typespec-azure-portal-core",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "tsc -p .",
+    "test": "vitest run",
+    "test:watch": "vitest -w",
+    "test-official": "vitest run --coverage --reporter=junit --reporter=default --no-file-parallelism"
```

```suggestion
    "clean": "rimraf ./dist ./temp",
    "build": "tsc -p .",
    "watch": "tsc -p . --watch",
    "lint-typespec-library": "tsp compile . --warn-as-error --import @typespec/library-linter --no-emit",
    "test": "vitest run",
    "test:watch": "vitest -w",
    "test:ui": "vitest --ui",
    "test-official": "vitest run --coverage --reporter=junit --reporter=default --no-file-parallelism",
     "lint": "eslint . --ext .ts --max-warnings=0",
    "lint:fix": "eslint . --fix --ext .ts",
    "regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../docs/libraries/openapi/reference"
```
missing a few scripts

---

added them :) thank you so much!